docs/source/tutorials/memory_optimizations.rst

```diff
@@ -168,25 +168,33 @@ Lower Precision Optimizers
 *What's going on here?*
 
 In addition to :ref:`reducing model and optimizer precision <glossary_precision>` during training, we can further reduce precision in our optimizer states.
-All of our single-device fine-tuning recipes support lower-precision optimizers from the `bitsandbytes <https://huggingface.co/docs/bitsandbytes/main/en/index>`_ library -
-a good place to start might be the ``AdamW8bit`` and ``PagedAdamW8bit`` optimizers, which we've tested our recipes with.
+All of our single-device fine-tuning recipes support lower-precision optimizers from the `torchao <https://github.com/pytorch/ao/tree/main/torchao/prototype/low_bit_optim>`_
+or `bitsandbytes <https://huggingface.co/docs/bitsandbytes/main/en/index>`_ libraries - a good place to start might be the ``AdamW8bit`` and ``PagedAdamW8bit`` optimizers,
+which we've tested our recipes with. Note that the implementation from bitsandbytes, at the moment this article was written, is not compatible with
+our distributed training recipes.
 
 *Sounds great! How do I use it?*
 
-To use this in your recipes, make sure you have installed bitsandbytes (``pip install bitsandbytes``). Then, enable
+To use this in your recipes, make sure you have installed torchao (``pip install torchao``) or bitsandbytes (``pip install bitsandbytes``). Then, enable
```

Comment:
Why not just show that users can do both? It's a good opportunity to show plug and play operability with the fine-tuning ecosystem.